src/components/SubmitAction/index.css

```diff
@@ -0,0 +1,25 @@
+.SubmitAction {
+  background-color: #81bc00;
+  border: 0;
+  box-shadow: 0 0.125rem #1d1d1d;
+  color: #ffffff;
+  font: 600 1.25rem/1.5rem Nova, sans-serif;
+  padding: 0.75rem;
+  text-align: center;
+  white-space: normal;
+  width: 100%;
+}
+
+.SubmitAction:hover {
+  background-color: color(#81bc00 shade(10%));
```

Comment:
Not seen `color(red shade(10%))` before. How is the support?

Reply:
Gatsby bundles [postcss-cssnext](http://cssnext.io/) by default. We're currently using it to transpile the `system-ui` generic font keyword. 

It [can also transpile](http://cssnext.io/features/#color-function) the `color()` function, an upcoming feature of the CSS langauge.